### PR TITLE
IRV: make charts interactive to each other and with the map

### PR DESCRIPTION
**Pull request by ptormene**
These are important steps towards a better interaction between map and charts and among charts (see https://github.com/gem/oq-platform/issues/596).

When a zone is clicked in the map, the corresponding line is toggled (highlighted or unhighlighted) in all charts. Consecutive clicks on different zones make all those zones highlighted in the charts.

When brushes are used to filter data in one chart, the selected zones are also highlighted in the other charts.

Tests are green: https://ci.openquake.org/job/zdevel_oq-platform/300/

### CODE REVIEW

File: openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js [hunk 1]
```diff
@@ -1084,19 +1102,28 @@ function mapboxGlLayerCreation() {
...
-                if (typeof region !== 'undefined') {
-                    $('#mapInfo').append('<h4>' + region + '</h4>');
-                }
+                region = features[0].properties.region;
             } catch(exc) {
                 // do not show info
             }
+            if (typeof region !== 'undefined') {
+                toggleRegionInCharts(region);
+                $('#mapInfo').append('<h4>' + region + '</h4>');
```
Comment (nastasi-oq) [inline openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js]: `.append($('<h4/>').html(region);)` is more robust.

**Comment by nastasi-oq** [comment]
Tests are missing.
Please, at least, open an issue on the bug-tracker to have a reminder about develop a test for the new features.

**Comment by ptormene** [comment]
As requested, I've added an issue (https://github.com/gem/oq-platform/issues/602) as a reminder for additional selenium tests for the new features.

File: openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js (hunk 1 above)
Comment (ptormene) [inline openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js]: Done

**Comment by nastasi-oq** [comment]
LGTM

**Pull request merged**